From d3466a04345b31dfc62d94fe994b40321a6418ec Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Thu, 29 Sep 2016 15:43:06 -0400
Subject: [PATCH 1/1] Zend/acinclude.m4: don't warn about >=bison-3.0.1
 versions.

The PHP configure script will emit a warning if it thinks your bison
version is insufficient:

  WARNING: This bison version is not supported for regeneration of
  the Zend/PHP parsers (found: 3.0, min: 204, excluded: 3.0).

However, there is an error in the test that causes it to treat all
3.0.x versions as if they were 3.0. The result is that users get
warned about a perfectly acceptable version of bison.

This patch is meant only for Gentoo, where we can require a working
version of bison (something newer than 3.0.1). Having done so, this
patch removes the check and the WARNING.

Gentoo-Bug: 593278
PHP-Bug: 69055
---
 Zend/acinclude.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Zend/acinclude.m4 b/Zend/acinclude.m4
index 7fa8c99..9d6cb1d 100644
--- a/Zend/acinclude.m4
+++ b/Zend/acinclude.m4
@@ -9,7 +9,7 @@ AC_DEFUN([LIBZEND_BISON_CHECK],[
   # non-working versions, e.g. "3.0 3.2";
   # remove "none" when introducing the first incompatible bison version an 
   # separate any following additions by spaces
-  bison_version_exclude="3.0"
+  bison_version_exclude=""
 
   # for standalone build of Zend Engine
   test -z "$SED" && SED=sed
-- 
2.7.3

